frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx

```diff
@@ -57,6 +57,31 @@ describe("EnterReceiptPage Component", () => {
     });
   });
 
+  it("removes row from table when selecting remove button", async () => {
+    const { getByPlaceholderText, getByText, container } = render(
+      <EnterReceiptPage />,
+      {
+        authContext: {
+          apiCall: fetchMock
+        }
+      }
+    );
+    const inputField = getByPlaceholderText("SDN");
+    const submit = getByText("Search");
+    fireEvent.change(inputField, {
+      target: { value: "wrongsdn" }
+    });
+    await wait(() => {
+      expect(inputField).toHaveValue("wrongsdn");
+    });
+    fireEvent.click(submit);
+    const removeIcon = container.querySelector("button.usa-button") as Element;
+    fireEvent.click(removeIcon);
+    await wait(() => {
+      expect(removeIcon).not.toBeInTheDocument();
+    });
```

Comment:
and then this assertion could expect the sdn to no longer be in the document

`expect(queryByText('wrongsdn')).not.toBeInTheDocument()`

Reply:
Is it better to use both assertions or replace with the new assertion?